src/app/(navigation-progammatically)/order/page.tsx

```
@@ -0,0 +1,20 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+
+type Props = {};
+
+export default function OrderPage({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)